discovery-service/src/main/java/org/zowe/apiml/discovery/GatewayNotifier.java

```diff
@@ -165,7 +169,10 @@ protected void serviceUpdatedProcess(String serviceId, String instanceId) {
         notify(instanceId, instanceInfo -> {
             final String url = getServiceUrl(serviceId, instanceInfo);
             try {
-                restTemplate.delete(url);
+                CloseableHttpResponse response = httpClient.execute(new HttpDelete(url));
+                if (response.getStatusLine().getStatusCode() >= 300) {
+                    throw new ClientProtocolException("Unexpected response: " + response.getStatusLine());
```

Comment:
a bit strange to throw an error in the catch block, can we log the response with the status code and catch only IOException with a separate log?